syntax = "proto3";

service SourceCodeParser {
    rpc SyntaxParse(stream SyntaxParseRequest) returns (stream SyntaxParseResponse);
}

message SyntaxParseRequest {
    string user_id = 1;
    string file_path = 2;
}

message SyntaxParseResponse {
    bool success = 1;
}

// python -m grpc_tools.protoc -I . --python_out=. --grpc_python_out=. source_code_parser/protobuf/source_code_parser.proto
